com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/StepLogger.java

```diff
@@ -92,6 +92,11 @@ public void error(String message) {
         sendProgressMessage(message, ProgressMessageType.ERROR);
     }
 
+    public void error(String message, Exception e, String tail) {
```

Comment:
Think about renaming tail to something more suitable.
E.g. supportMessageHint

If the intention is to use for more generic purpose and can be extend further, it can be named with "messageDetails" or "messageHints"